assessor/tests.py

```diff
@@ -0,0 +1,26 @@
+from django.test import TestCase
+from rest_framework.reverse import reverse
+from rest_framework.test import APIClient
+from users.models import OdiUser
+import json
+
+
+class ViewsTestCase(TestCase):
+    def setUp(self):
```

Comment:
Implementation of test is sufficient for the current situation as there is still no interaction with the database. However, please add the tests for all functions that contributes to the dashboard display functionality. Ensure to use mocking and stubbing to ease the testing process.